plugins/c8y_firmware_plugin/src/lib.rs

```diff
@@ -113,6 +113,7 @@ async fn run_with(tedge_config: TEdgeConfig) -> Result<(), anyhow::Error> {
         &mut mqtt_actor,
         &mqtt_schema,
         tedge_config.service.ty.clone(),
+        tedge_config.mqtt.topic_root.clone(),
```

Comment:
The same argument is given twice: first as an `MqttSchema`, second as `String`. Keep only the first one.

Reply:
fixed